### DISCUSSION TITLE
SRS newbie questions

### DISCUSSION THREAD

**Discussion by Pixouls** [st3v3nmw/obsidian-spaced-repetition#158] [Q&A] [answered]
Hello! I'm new to SRS and am trying to learn to use my Obsidian vault more effectively. After looking at the other SRS plugins, I decided that this might best fit my specs the best.

I'm currently studying for a LEED exam. Some things that come up are charts of how many points each requirement gets, vocab, descriptions of requirements, and practice questions. Right now, I've just thrown everything into Quizlet, but I'm trying to think about how I can turn things into an organized Obsidian vault so I can have a wiki of all this information and also use SRS to help me revise as the exam approaches. 

Some of my basic questions are:
Is it possible to have flashcards be generated from within a note that has other things in it? Like say I have a page about water quality and I want to put all the flashcards about water quality relevant to that section on the bottom. Or can flashcards only be generated from relevant flashcard note files? 

Another question I have is whether flashcards support backlinks. For example:
Q: What are the main requirements for water quality?
A: [[Requirement 1]], [[Requirement 2]], [[Requirement 3]]

Sorry if these seem like confusing questions, I'm new to tools like these so I'm not completely picking up how I would apply this tool to my use case yet or if I should change my approach more to work better with this tool. Most people I see using SRS in general are using it to study for language which is not what I want to apply this to at all. I've been slowly going through all the threads on the forum and issues and discussions but it's taking me a lot of time and well, I need to study too haha.

I'm also thinking about how to balance the wiki approach with the flashcard making. For example, say I make a note for each vocab term or requirement so that when those terms come up in the larger note sheets, I can hover to see the vocab word definition. However, it seems like I would need to do that and make a separate note card file of all the vocab if I also wanted to use SRS to go over them. Maybe the move would be to do a link to the block on the flashcard page where I have the definition, rather than making a new note for each vocab word? But then I wouldn't see which vocab words are getting the most links.

**Comment by st3v3nmw** [comment, marked as answer]
Hello!

>  Is it possible to have flashcards be generated from within a note that has other things in it? Like say I have a page about water quality and I want to put all the flashcards about water quality relevant to that section on the bottom. Or can flashcards only be generated from relevant flashcard note files?

Yes, you can mix everything up. Depending on your preferences, you can tag a file using tags like `#flashcards` to tell the plugin to look for flashcards in that particular file or you can have the plugin scan all files and automatically find the ones that have flashcards. Like:
```markdown
Some notes

flashcard i.e. an inline one like question::answer

More notes

```

> Another question I have is whether flashcards support backlinks. For example:
> Q: What are the main requirements for water quality?
> A: [[Requirement 1]], [[Requirement 2]], [[Requirement 3]]

No, this is not currently supported. The links will appear but they won't work when clicked. That limitation is inbuilt into Obsidian since the `renderMarkdown` function Obsidian exposes to plugins doesn't support that. At least, not yet.

> I'm also thinking about how to balance the wiki approach with the flashcard making. For example, say I make a note for each vocab term or requirement so that when those terms come up in the larger note sheets, I can hover to see the vocab word definition. However, it seems like I would need to do that and make a separate note card file of all the vocab if I also wanted to use SRS to go over them. Maybe the move would be to do a link to the block on the flashcard page where I have the definition, rather than making a new note for each vocab word? But then I wouldn't see which vocab words are getting the most links.

With Obsidian, I think it's better to use the block references for the vocab bit.
Otherwise you can use atomic notes where each note represents one idea. That's my current approach; atomic notes, & lots of linking between 'em.

**Reply by Pixouls** [reply]
> Yes, you can mix everything up. Depending on your preferences, you can tag a file using tags like #flashcards to tell the plugin to look for flashcards in that particular file or you can have the plugin scan all files and automatically find the ones that have flashcards. 

Wow, I'm so excited to try this out, thank you!

>  block references for the vocab bit

i ended up starting to make atomic notes, but I'm not completely sold on it. maybe i will switch to block quotes, i will work on that a bit today and let you know once I've made progress.

**Reply by Pixouls** [reply]
I realize one issue with using block quotes for definitions is that if it is multiline, I can only link to one block, so I would not be able to see the whole definition on hover. 

would it be possible to have the card be

```
# question
?
answer
```
that way i can link to the heading. I'm not sure how to refresh cards to test if this works

update: just tried it, seems like it works
